phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixStatement.java

```diff
@@ -1248,8 +1248,8 @@ public MutationState execute() throws SQLException {
 
     private static class ExecutableAlterIndexStatement extends AlterIndexStatement implements CompilableStatement {
 
-        public ExecutableAlterIndexStatement(NamedTableNode indexTableNode, String dataTableName, boolean ifExists, PIndexState state, boolean async, ListMultimap<String,Pair<String,Object>> props) {
-            super(indexTableNode, dataTableName, ifExists, state, async, props);
+        public ExecutableAlterIndexStatement(NamedTableNode indexTableNode, String dataTableName, boolean ifExists, PIndexState state, boolean isAll, boolean async, ListMultimap<String,Pair<String,Object>> props) {
+            super(indexTableNode, dataTableName, ifExists, state, isAll, async, props);
```

Comment:
Let's keep the param name in-sync between the jdbc classes and the parser classes - isAll here and isRebuildAll below. Let's pick one. I'd suggest 'rebuildAll' (to keep it consistent everywhere and inline with async without [is]).

Reply:
This is auto generated. I don't think I can get rid of 'is' part

Reply:
Also, I prefer isRebuildAll because this is just returning the boolean parameter. However, rebuildAll suggests a function name where you actually rebuild the index. It is harder to read the code that way.